paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -1056,6 +1056,12 @@ public class CoreOptions implements Serializable {
                                     + "If the data size allocated for the sorting task is uneven,which may lead to performance bottlenecks, "
                                     + "the config can be set to size.");
 
+    public static final ConfigOption<Integer> SORT_COMPACTION_LOCAL_SAMPLE_SIZE =
+            key("sort-compaction.local-sample.size")
+                    .intType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "The size of local sample for sort-compaction.By default,the value is the sink parallelism * 1000.");
```

Comment:
I think the parameter could be magnification.  int localSampleSize = parallelism * magnification.   magnification should be greater than or equal to 20